uint64 maps to unsigned long long

sloretz opened the pull request:
Previously article mapped uint64 to `unsigned long` which is synonymous with `unsigned int`.

sloretz committed:
uint64 maps to unsigned long long

Previously article mapped uint64 to `unsigned long` which is synonymous with `unsigned int`.

mjcarroll commented on articles/142_idl.md:
```diff
@@ -99,7 +99,7 @@ the DDS-XTypes specification 1.2 defines it with 16-bit.
 | int32          | int                 |
 | uint32         | unsigned int        |
 | int64          | long                |
```
Should `int64` be correspondingly `long long`

sloretz replied:
3b9f617

dirk-thomas commented:
The `(u)int32` types should also be changed to `(unsigned) long`.

sloretz committed:
long -> long long + int -> long

(u)int32 types are called `long` not `int` in IDL. Also `int64` is `long long` not `long`

mjcarroll reviewed (changes requested):
Rescinding my previous approval ;)

sloretz commented:
> The (u)int32 types should also be changed to (unsigned) long.

3b9f617
